src/setup/cortx_setup.py

```diff
@@ -115,6 +114,38 @@ def process(self, *args, **kwargs):
         return 0
 
 
+class UpgradeCmd(Cmd):
+    """ Cluster Upgrade Cmd """
+
+    name = 'upgrade'
+
+    def __init__(self, args: dict):
+        super().__init__(args)
+
+    @staticmethod
+    def add_args(parser: str):
+        """ Add Command args for parsing """
+
+        parser.add_argument('-c', dest='cortx_conf', help='Cortx Config URL')
```

Comment:
Command syntax is 
cortx_setup <noun> <verb> <parameters>
e.g. cortx_setup cluster bootstrap ... 

This is broken. We need to maintain the structure. Example..
cortx_setup upgrade {start|prepare|...} <parameters> 
OR 
cortx_setup cluster upgrade 
OR 
similar

Reply:
changed cmd syntax to:
 cortx_setup cluster upgrade